libraries/slave_jnlp.rb

```diff
@@ -54,19 +54,18 @@ def load_current_resource
 
     def action_create
       super
-
-      parent_remote_fs_dir_resource.run_action(:create)
-
-      # don't create user/group on Windows
+    
```

Comment:
@ksubrama Thought it was easier to understand what was going on.  Usually if I see a bunch of conditional statements in a class, and we already have a windows specific class,  thought it would make it easier to understand.